tools/challenge-parser/package.json

```diff
@@ -1,14 +1,25 @@
 {
   "name": "challenge-parser",
```

Comment:
```suggestion
  "name": "@freecodecamp/challenge-parser",
```
Is that the rename you wanted?  If so, yep, I'm totally onboard with that.

Reply:
I happened across https://github.blog/2021-02-12-avoiding-npm-substitution-attacks/ today, so I see the need!

Reply:
This must have been a happy accident in that case.